local.yml

```diff
@@ -7,5 +7,4 @@
     is_container: false
   roles:
   - role: "{{ playbook_dir }}"
-    rhel7stig_system_is_container: "{{ is_container | default(false) }}"
-    rhel7stig_ssh_required: "{{ is_container == false  }}"
```

Comment:
This `rhel7cis_ssh_required` and `rhey7stig_ssh_required` is not referenced anywhere in the actual role, so i removed it.